socorro/tests/external/test_crashstorage_base.py

```diff
@@ -358,51 +155,3 @@ def test_simple(self):
         fdm = mdm.as_file_dumps_mapping("a", "/tmp", "dump")
         assert fdm.as_file_dumps_mapping() is fdm
         assert fdm.as_memory_dumps_mapping() == mdm
-
-
-class TestMetricsCounter:
```

Comment:
This metrics counter acts as a crashstorage destination. Instead of the complexity around that, it's easier to add a `METRICS.incr` call in the processor.